src/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -1182,6 +1182,9 @@
   <data name="InvalidProgram_CallVirtFinalize" xml:space="preserve">
     <value>Object.Finalize() can not be called directly. It is only callable by the runtime.</value>
   </data>
+  <data name="InvalidProgram_NativeCallable" xml:space="preserve">
+    <value>Common Language Runtime detected an invalid program. NativeCallable method cannot be called from managed code.</value>
```

Comment:
Nit: Omit `Common Language Runtime detected an invalid program.` The other specific `InvalidProgram_...` messages do not have it either.